zenith/src/main.rs

```diff
@@ -488,86 +529,92 @@ fn handle_pg(pg_match: &ArgMatches, env: &local_env::LocalEnv) -> Result<()> {
     let mut cplane = ComputeControlPlane::load(env.clone())?;
 
     // All subcommands take an optional --tenantid option
-    let tenantid = get_tenantid(sub_args, env)?;
+    let tenant_id = get_tenant_id(sub_args, env)?;
 
     match sub_name {
         "list" => {
-            let branch_infos = get_branch_infos(env, &tenantid).unwrap_or_else(|e| {
-                eprintln!("Failed to load branch info: {}", e);
+            let timeline_infos = get_timeline_infos(env, &tenant_id).unwrap_or_else(|e| {
+                eprintln!("Failed to load timeline info: {}", e);
                 HashMap::new()
             });
 
-            println!("NODE\tADDRESS\t\tBRANCH\tLSN\t\tSTATUS");
+            println!("NODE\tADDRESS\t\tTIMELINE\tLSN\t\tSTATUS");
             for ((_, node_name), node) in cplane
                 .nodes
                 .iter()
-                .filter(|((node_tenantid, _), _)| node_tenantid == &tenantid)
+                .filter(|((node_tenantid, _), _)| node_tenantid == &tenant_id)
             {
                 // FIXME: This shows the LSN at the end of the timeline. It's not the
                 // right thing to do for read-only nodes that might be anchored at an
                 // older point in time, or following but lagging behind the primary.
-                let lsn_str = branch_infos
-                    .get(&node.timelineid)
+                let lsn_str = timeline_infos
+                    .get(&node.timeline_id)
                     .map(|bi| bi.latest_valid_lsn.to_string())
                     .unwrap_or_else(|| "?".to_string());
 
                 println!(
                     "{}\t{}\t{}\t{}\t{}",
                     node_name,
                     node.address,
-                    node.timelineid, // FIXME: resolve human-friendly branch name
+                    node.timeline_id,
                     lsn_str,
                     node.status(),
                 );
             }
         }
         "create" => {
             let node_name = sub_args.value_of("node").unwrap_or("main");
-            let timeline_name = sub_args.value_of("timeline").unwrap_or(node_name);
```

Comment:
This is another place where we're substituting a `timeline` that is already a `branch_name` and a `node_name` for a typical test of ours due to https://github.com/zenithdb/zenith/pull/1286/files#r807832063